FinSearchUnified/Tests/Bundle/ProductNumberSearchTest.php

```diff
@@ -264,6 +269,96 @@ public function testProductNumberSearchResultWithAllFilters(SimpleXMLElement $xm
         }
     }
 
+    public function selectedFiltersProvider()
```

Comment:
Since we had some issues that caused selected filters not to work, add a test where the facet is selected but has no items.